src/delfic3/models.py

```diff
@@ -1,4 +1,21 @@
+"""
+Models for Delfic3 mission.
+"""
+import datetime
+from django.db import models
+from ewilgs.models import Downlink
 
-# from django.db import models
-
-# Create your models here.
+class Delfic3_L0_telemetry(models.Model): #pylint: disable=C0103
+    """
+    Telemetry Table Delfic3
+    """
+    id = models.ForeignKey(Downlink, primary_key=True, editable=False, on_delete=models.DO_NOTHING)
+    frame_time = models.IntegerField(default=None, null=True)
+    send_time = models.IntegerField(default=None, null=True)
+    receive_time = models.TimeField(null=False, default = datetime.time )
```

Comment:
I would switch frame_time and send_time to TimeFiels (as receive_time) mostly because visualization will be easier. It's not really a strong requirement but I feel it will make our life easier later (not bothering about turning a number of ms since 1970 into a date manually). For the system, there will not be a difference and I am not scared of converting all values when importing the tables.